### ISSUE TITLE
Citation should not be mandatory

### ISSUE THREAD

**Issue by jonathanthiry** [eu-interact/data-guidelines#12]
Infrastructure datasets do not often have academic articles linked to them and cannot provide any citation.

**Comment by mikkolata** [comment]
I agree with this one. For our datasets which are not fixed but are been updated on daily basis, there isnt any articles or publications.

Though I can always just leave it as null, NaN or any other value which indicates that there isnt data available on that one but it still exists as metadata.
